ui/app/AppLayouts/Chat/ContactsColumn/Channel.qml

```diff
@@ -9,9 +9,13 @@ Rectangle {
     property string lastMessage: "My latest message\n with a return"
     property string timestamp: "20/2/2020"
     property string unviewedMessagesCount: "2"
+    property bool hasMentions: false
     property int chatType: Constants.chatTypePublic
     property string searchStr: ""
 
+    Component.onCompleted: {
+      console.log(name + ": " + hasMentions)
+    }
```

Comment:
@richard-ramos so I see this being logged once with `hasMentions` being false, however I also see `hasMentions` being updated to `true` for the dedicated channel, but it's not reflected in the UI.

I've introduced a `QtProperty` with a getter but still no luck. Is there anything else that needs to be done to have `hasMentions` state of each channel properly reflected in the UI?

Reply:
Found the issue. Needed to update `dataChanged` in `ChannelList`